### PR TITLE
[BE-03] Add level field to recipes 

### PR DESCRIPTION
**Pull request by pannoewo**
AC:
1. level field may be set during create or update a record
2. level field is returned in #show response.
3. level field has value 1 - 5
4. level field returns ‘unknow’ if value is empty.

https://trello.com/c/mSpUFAn4

### CODE REVIEW

File: app/controllers/api/v1/recipes_controller.rb [hunk 1]
```diff
@@ -15,30 +15,30 @@ def create
         if @recipe.save
           render json: RecipeShowSerializer.new(@recipe).to_h
         else
-          render json: { errors: @recipe.errors.to_s }, status: :unprocessable_entity
+          render json: { errors: @recipe.errors.full_messages }, status: :unprocessable_entity
```
Comment (plokijro) [inline app/controllers/api/v1/recipes_controller.rb:18]: 👌

File: spec/controllers/api/v1/recipes_controller_spec.rb [hunk 2]
```diff
@@ -108,8 +109,10 @@
     let(:new_cooking_time) { 1000 }
     let(:old_video_link) { 'https://www.google.pl/' }
     let(:new_video_link) { 'https://guides.rubyonrails.org/index.html' }
+    let(:old_level) { 3 }
+    let(:new_level) { 1 }
     let(:recipe) do
-      create(:recipe, name: old_name, content: 'asdas', cooking_time: old_cooking_time, video_link: old_video_link)
+      create(:recipe, name: old_name, content: 'asdas', cooking_time: old_cooking_time, video_link: old_video_link, level: old_level)
```
Comment (plokijro) [inline spec/controllers/api/v1/recipes_controller_spec.rb:115]: Too long line

**Review by plokijro — APPROVED** [review]

File: spec/controllers/api/v1/recipes_controller_spec.rb (hunk 2 above)
Comment (pannoewo) [reply spec/controllers/api/v1/recipes_controller_spec.rb:115]: what can i do

Comment (pannoewo) [reply spec/controllers/api/v1/recipes_controller_spec.rb:115]: enter tak o po prostu?

**Review by grabomarcin — APPROVED** [review]

File: spec/controllers/api/v1/recipes_controller_spec.rb (hunk 2 above)
Comment (grabomarcin) [reply spec/controllers/api/v1/recipes_controller_spec.rb:115]: xd

Comment (plokijro) [reply spec/controllers/api/v1/recipes_controller_spec.rb:115]: 😁

File: spec/controllers/api/v1/recipes_controller_spec.rb [hunk 3]
```diff
@@ -164,7 +167,27 @@
...
+      let(:recipe_put_level) do
+        {
+          id: recipe.id,
+          recipe: { level: new_level }
+        }
+      end 
+      
+      it 'updated level' do
+        expect do
+          put :update, params: recipe_put_level
```
Comment (DanielOldak) [inline spec/controllers/api/v1/recipes_controller_spec.rb:181]: You can extract ``` put :update, params: recipe_put_level ``` to let, something like
```let(:update_request) { put :update, params: recipe_put_level }```

and call it with expect{ update_request }.to ...

**Pull request merged**